pkg/daemon/update.go

```diff
@@ -155,6 +155,7 @@ func (dn *Daemon) performPostConfigChangeAction(postConfigChangeActions []string
 	}
 
 	// currentConfig != desiredConfig, kick off an update
+	// TODO: this is recursive back to update(),  and now that we do rebootless updates we need to think about this
```

Comment:
This function will usually get called if something in between got changed that lead to a generation of a new config like a new MachineConfig got applied while update was already in progress. This is to make sure that we always update node to desired config.

In regular flow, following will be called.
```
if inDesiredConfig {
		return nil
	}
```

Reply:
Not sure if we really need this comment being added.

Reply:
would prefer card over in-code to-dos, agree with sinny to remove

Reply:
Thanks, todo has been removed.